common/views/components/License/LicenseBeta.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import { Fragment } from 'react';
+import getLicenseInfo from '../../../utils/get-license-info';
+import type { LicenseType } from '../../../model/license';
+import type { LicenseData } from '../../../utils/get-license-info';
+
+type Props = {|
+  subject: string,
+  licenseType: LicenseType,
+|};
+
+const LicenseBeta = ({ subject, licenseType }: Props) => {
```

Comment:
Just thinking more about `beta` - seems it might come into conflict with the other weird thing we call beta. No problem with me though,